module-presentation/src/main/java/com/depromeet/friend/api/FollowController.java

```diff
@@ -0,0 +1,67 @@
+package com.depromeet.friend.api;
+
+import com.depromeet.config.Logging;
+import com.depromeet.dto.response.ApiResponse;
+import com.depromeet.friend.dto.request.FollowingRequest;
+import com.depromeet.friend.dto.response.FollowSliceResponse;
+import com.depromeet.friend.dto.response.FollowerFollowingCountResponse;
+import com.depromeet.friend.dto.response.FollowerResponse;
+import com.depromeet.friend.dto.response.FollowingResponse;
+import com.depromeet.friend.facade.FollowFacade;
+import com.depromeet.member.annotation.LoginMember;
+import com.depromeet.type.friend.FollowSuccessType;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/friend")
+public class FollowController implements FollowApi {
+    private final FollowFacade followFacade;
+
+    @PostMapping
+    @Logging(item = "Following/Follower", action = "POST")
```

Comment:
```suggestion
    @Logging(item = "Follower/Following", action = "POST")
```
순서 통일하는 게 좋을 것 같아요
<img width="309" alt="Screenshot 2024-08-14 at 15 07 47" src="https://github.com/user-attachments/assets/ed7e6dda-af9d-409e-955e-3437e3c24c4b">

Reply:
수정 완료